migration/migration.go

```diff
@@ -272,7 +272,7 @@ func InitSchemas(ctx context.Context, to, from DBConn) (err error) {
 		return
 	}
 	defer postgres.Disconnect(ctx)
-
+	reqLogger.Info("This has InsecureSkipVerify changes")
```

Comment:
I'd rather not log this out.

Reply:
Sure added this log as part of testing, will be removing this log thank you